SignalServiceKit/src/Devices/OWSDevice.h

```diff
@@ -52,6 +52,11 @@ extern uint32_t const OWSDevicePrimaryDeviceId;
  */
 - (BOOL)updateAttributesWithDevice:(OWSDevice *)other;
 
+#pragma mark - "May Have Linked Devices" Flag
+
++ (BOOL)mayHaveLinkedDevices:(YapDatabaseConnection *)dbConnection;
```

Comment:
nit: This isn't broken, but it does not seem like a a model level concern. e.g. belongs in a manager or other global singleton.

Reply:
Done.